extensions-contrib/prometheus-emitter/pom.xml

```diff
@@ -43,17 +43,17 @@
     <dependency>
       <groupId>io.prometheus</groupId>
       <artifactId>simpleclient</artifactId>
-      <version>0.7.0</version>
+      <version>0.16.0</version>
```

Comment:
Thanks!

I assume this version is fully backward compatible and doesn't cause any upgrade issues.